01_prepare_host.sh

```diff
@@ -6,16 +6,18 @@ source lib/logging.sh
 # shellcheck disable=SC1091
 source lib/common.sh
 
-sudo modprobe -r -a kvm_intel kvm
-sudo modprobe kvm tdp_mmu=0
-sudo modprobe -a kvm kvm_intel
-
 if [[ $(id -u) == 0 ]]; then
   echo "Please run 'make' as a non-root user"
   exit 1
 fi
 
 if [[ $OS == ubuntu ]]; then
```

Comment:
are you going to change this check to point to ubuntu version or shall we take it as it is?

Reply:
We will do that in a follow up

Reply:
/hold 
I will do it here

Reply:
ok 
/approve